src/main/java/com/stripe/model/StripeCollectionAPIResource.java

```diff
@@ -33,6 +34,9 @@ public void setHasMore(Boolean hasMore) {
 	public String getURL() {
 		return url;
 	}
+	public String getUrl() {
+		return url;
+	}
```

Comment:
Can we deprecate one of `getURL` or `getUrl` them, and have the deprecated one call the non-deprecated one?